packages/mesos-modules/build

```diff
@@ -1,15 +1,53 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
+# Update the image and download build tools.
+apt-get update -qq
+apt-get install -y autoconf libtool make pkg-config
```

Comment:
This sort of thing has to happen in a docker build, it is not reliable / re-producible, which means this package build with it is not reproducible.

The dco-ca package has an example of this currently: https://github.com/mesosphere/dcos-enterprise/tree/master/packages/dcos-ca. (Packages can define a Dockerfile to build the build env for that package)

The other option is to add it to the base DC/OS package build docker.

The docker is built once, and as long as the Dockerfile doesn't change forever re-used, so that individual package builds are hopefully deterministic.

Ideally for these though we _really_ need to use the standard / same docker we use for building:
1. mesos dependencies
2. mesos
3. mesos-modules

If those dockers differ, than odd incompatibilities and crashes start showing up.